.changeset/honest-bikes-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": minor
+---
+
+Added the new rule [useSingleJsDocAsterisk](https://biomejs.dev/linter/rules/use-single-js-doc-asterisk/) which enforces JSDoc comment lines to start with a single asterisk.
```

Comment:
This is a new feature, we should explain it to the user, maybe with a nice code snippet that shows when the rule is triggered. I believe the explanation is fine, we just need to add a code block